webhooks-extension/README.md

```diff
@@ -5,6 +5,49 @@ The Webhooks Extension for Tekton provides allows users to set up Git webhooks t
 
 In addition to Tekton/Knative Eventing glue, it includes an extension to the Tekton dashboard.
 
+## Install
+
+- set `DOCKER_REPO` with your docker repository in `build.sh`, `config/extension-deployment.yaml`, and `config/sink-kservice.yaml`.
+- run `./build.sh`
+- set the `INSTALL_NAMESPACE` in the `config/extension-deployment.yaml` and `config/sink-kservice.yaml` files
+- run `kubectl apply -f config/ -n INSTALL_NAMESPACE`, where `INSTALL_NAMESPACE` is the same as the `INSTALL_NAMESPACE` set in the previous step
+
+## Endpoints
+
```

Comment:
I wanted to document the API endpoints that each pod has. Since there are two different pods, I wanted to separate the documentation for each of them; so I separated it into two files. However, I can make separate sections in the main README, if you think that would be preferable @dibbles ?